polytracker/src/CMakeLists.txt

```diff
@@ -50,7 +35,21 @@ set(TAINTDAG_SOURCES
     ${TAINTDAG_DIR}/fnmapping.cpp
     ${TAINTDAG_DIR}/fntrace.cpp)
 
-add_library(Polytracker STATIC ${POLY_SOURCES} ${TAINT_SOURCES}
+add_library(Polytracker STATIC ${POLYTRACKER_SOURCES} ${TAINT_SOURCES}
                                ${TAINTDAG_SOURCES})
+
+set(POLYTRACKER_COMPILE_OPTS -fno-exceptions -fno-rtti -Wall -Werror
+                             -stdlib=libc++ -fPIC)
```

Comment:
Just making sure, the `set(CMAKE_POSITION_INDEPENDENT_CODE ON)` is replaced by the `-fPIC`, right?

Reply:
Yes.